MdePkg/Include/Library/DebugLib.h

```diff
@@ -19,12 +19,14 @@ SPDX-License-Identifier: BSD-2-Clause-Patent
 //
 // Declare bits for PcdDebugPropertyMask
 //
-#define DEBUG_PROPERTY_DEBUG_ASSERT_ENABLED       0x01
-#define DEBUG_PROPERTY_DEBUG_PRINT_ENABLED        0x02
-#define DEBUG_PROPERTY_DEBUG_CODE_ENABLED         0x04
-#define DEBUG_PROPERTY_CLEAR_MEMORY_ENABLED       0x08
-#define DEBUG_PROPERTY_ASSERT_BREAKPOINT_ENABLED  0x10
-#define DEBUG_PROPERTY_ASSERT_DEADLOOP_ENABLED    0x20
+#define DEBUG_PROPERTY_DEBUG_ASSERT_ENABLED        0x01
+#define DEBUG_PROPERTY_DEBUG_PRINT_ENABLED         0x02
+#define DEBUG_PROPERTY_DEBUG_CODE_ENABLED          0x04
+#define DEBUG_PROPERTY_CLEAR_MEMORY_ENABLED        0x08
+#define DEBUG_PROPERTY_ASSERT_BREAKPOINT_ENABLED   0x10
+#define DEBUG_PROPERTY_ASSERT_DEADLOOP_ENABLED     0x20
+#define DEBUG_PROPERTY_ASSERT_BREAKASSERT_ENABLED  0x40 // MU_CHANGE BREAKASSERT
+#define DEBUG_PROPERTY_ASSERT_TELEMETRY_ENABLED    0x80 // MU_CHANGE LOGTELEMETRY

```

Comment:
This this be coming in a different commit?

Reply:
Are you saying this should be in a separate commit/split it in two?

Reply:
@kuqin12 , LOGTELEMETRY doesn't get used anywhere, is that because the expectation is for a platform to implement it?

Reply:
This macro is used in AdvLoggerPkg: https://github.com/microsoft/mu_plus/blob/90f8ca98e2a7c7bc22a60b6e9fe18c69ce57d92f/AdvLoggerPkg/Library/AssertTelemetryLib/AssertLib.c#L95